2021-07-02 15:44:56,827 INFO    MainThread:17176 [wandb_setup.py:_flush():69] setting env: {}
2021-07-02 15:44:56,827 INFO    MainThread:17176 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-02 15:44:56,827 INFO    MainThread:17176 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-02 15:44:56,828 INFO    MainThread:17176 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\wandb\run-20210702_154456-2w66s6wl\logs\debug.log
2021-07-02 15:44:56,828 INFO    MainThread:17176 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\wandb\run-20210702_154456-2w66s6wl\logs\debug-internal.log
2021-07-02 15:44:56,830 INFO    MainThread:17176 [wandb_init.py:_jupyter_setup():288] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001AA6B9809B0>
2021-07-02 15:44:56,830 INFO    MainThread:17176 [wandb_init.py:init():370] calling init triggers
2021-07-02 15:44:56,830 INFO    MainThread:17176 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-02 15:44:56,830 INFO    MainThread:17176 [wandb_init.py:init():392] re-initializing run, found existing run on stack: 2z83buwz
2021-07-02 15:44:56,833 INFO    MainThread:17176 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Project_gradient_boosting/2z83buwz
2021-07-02 15:44:56,834 INFO    MainThread:17176 [wandb_run.py:_atexit_cleanup():1594] got exitcode: 0
2021-07-02 15:44:56,835 INFO    MainThread:17176 [wandb_run.py:_restore():1566] restore
2021-07-02 15:44:57,757 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103826
  total_bytes: 103826
}

2021-07-02 15:44:57,878 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103826
  total_bytes: 103826
}

2021-07-02 15:44:58,197 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103826
  total_bytes: 103826
}

2021-07-02 15:44:59,094 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103826
  total_bytes: 107911
}

2021-07-02 15:44:59,442 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103826
  total_bytes: 107911
}

2021-07-02 15:44:59,545 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107911
  total_bytes: 107911
}

2021-07-02 15:44:59,647 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107911
  total_bytes: 107911
}

2021-07-02 15:44:59,750 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107911
  total_bytes: 107911
}

2021-07-02 15:44:59,852 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107911
  total_bytes: 107911
}

2021-07-02 15:44:59,954 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107911
  total_bytes: 107911
}

2021-07-02 15:45:00,055 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107911
  total_bytes: 107911
}

2021-07-02 15:45:00,159 INFO    MainThread:17176 [wandb_run.py:_wait_for_finish():1716] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107911
  total_bytes: 107911
}

2021-07-02 15:45:01,292 INFO    MainThread:17176 [wandb_run.py:_show_summary():1871] rendering summary
2021-07-02 15:45:01,294 INFO    MainThread:17176 [wandb_run.py:_show_history():1909] rendering history
2021-07-02 15:45:01,296 INFO    MainThread:17176 [wandb_run.py:_show_files():1938] logging synced files
2021-07-02 15:45:01,309 INFO    MainThread:17176 [wandb_init.py:init():419] starting backend
2021-07-02 15:45:01,309 INFO    MainThread:17176 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-07-02 15:45:01,317 INFO    MainThread:17176 [backend.py:ensure_launched():135] starting backend process...
2021-07-02 15:45:01,482 INFO    MainThread:17176 [backend.py:ensure_launched():140] started backend process with pid: 17316
2021-07-02 15:45:01,485 INFO    MainThread:17176 [wandb_init.py:init():424] backend started and connected
2021-07-02 15:45:01,513 INFO    MainThread:17176 [wandb_run.py:_label_probe_notebook():845] probe notebook
2021-07-02 15:45:01,513 INFO    MainThread:17176 [wandb_run.py:_label_probe_notebook():852] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-02 15:45:01,513 INFO    MainThread:17176 [wandb_init.py:init():472] updated telemetry
2021-07-02 15:45:01,515 INFO    MainThread:17176 [wandb_init.py:init():491] communicating current version
2021-07-02 15:45:04,476 INFO    MainThread:17176 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.10.33 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-07-02 15:45:04,478 INFO    MainThread:17176 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-02 15:45:04,918 INFO    MainThread:17176 [wandb_init.py:init():529] starting run threads in backend
2021-07-02 15:45:05,711 INFO    MainThread:17176 [wandb_run.py:_console_start():1624] atexit reg
2021-07-02 15:45:05,713 INFO    MainThread:17176 [wandb_run.py:_redirect():1498] redirect: SettingsConsole.WRAP
2021-07-02 15:45:05,713 INFO    MainThread:17176 [wandb_run.py:_redirect():1535] Wrapping output streams.
2021-07-02 15:45:05,716 INFO    MainThread:17176 [wandb_run.py:_redirect():1559] Redirects installed.
2021-07-02 15:45:05,716 INFO    MainThread:17176 [wandb_init.py:init():554] run started, returning control to user process
2021-07-02 15:45:09,260 INFO    MainThread:17176 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'outlier_candidates') {'id': 'wandb/outliers/v1', 'historyFieldSettings': {'key': 'outlier_candidates', 'x-axis': '_step'}} None
2021-07-02 15:45:59,157 INFO    MainThread:17176 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'learning_curve') {'id': 'wandb/learning_curve/v1', 'historyFieldSettings': {'key': 'learning_curve', 'x-axis': '_step'}} None
2021-07-02 15:46:05,904 INFO    MainThread:17176 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'residuals') {'id': 'wandb/residuals_plot/v1', 'historyFieldSettings': {'key': 'residuals', 'x-axis': '_step'}} None
2021-07-02 15:46:21,062 INFO    MainThread:17176 [wandb_setup.py:_flush():69] setting env: {}
2021-07-02 15:46:21,062 INFO    MainThread:17176 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-02 15:46:21,062 INFO    MainThread:17176 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-02 15:46:21,063 INFO    MainThread:17176 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\wandb\run-20210702_154621-1l1rb379\logs\debug.log
2021-07-02 15:46:21,063 INFO    MainThread:17176 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\wandb\run-20210702_154621-1l1rb379\logs\debug-internal.log
2021-07-02 15:46:21,065 INFO    MainThread:17176 [wandb_init.py:init():370] calling init triggers
2021-07-02 15:46:21,065 INFO    MainThread:17176 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-02 15:46:21,065 INFO    MainThread:17176 [wandb_init.py:init():392] re-initializing run, found existing run on stack: 2w66s6wl
2021-07-02 15:46:21,068 INFO    MainThread:17176 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Project_gradient_boosting/2w66s6wl
